### ISSUE TITLE
Import Settings?

### ISSUE THREAD

**Issue by mike-rowley** [volotat/SD-CN-Animation#98]
Very cool extension especially with the new Reference Only preprocessor.

I can see Export Settings, which is great but not very usable without a matching Import Settings.  Is that on the roadmap somewhere?  This would be a massive workflow improvement.

**Comment by volotat** [comment]
Yes, it is. I will try to bring it to the next version, but no promises yet. The main idea of exporting settings was to share them with others easily.

**Comment by mike-rowley** [comment]
Excellent, looking forward to it!  Thanks for building this!

**Comment by JulianVolodia** [comment]
@volotat any help appreciated? some milestones or design written?
